code/game/objects/items/food/bread.dm

```diff
@@ -37,6 +38,25 @@
 	. = ..()
 	AddElement(/datum/element/dunkable, 10)
 
+/obj/item/food/bread/add_context(atom/source, list/context, obj/item/held_item, mob/user)
```

Comment:
This should be on the component, not copied and pasted across all atoms that use the component

(Which is simple enough, you just need to register the add context signal via component instead of in the atom's init)

Reply:
I'm not honestly sure how to do that, based on other implementations it was done per item, not in a component. Is it just the same process but on `/datum/component/food_storage/` instead?